### PR TITLE
[Validator] Added missing translations for Czech (cs)

### LINKED ISSUE #51936: Missing translations for Czech (cs)
Hello,

There are some missing Czech (cs) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Czech (cs) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by shubhalgupta**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #51936
| License       | MIT

### CODE REVIEW

**Comment by shubhalgupta** [comment]
@carsonbot now please review!

**Comment by Nyholm** [comment]
Hey. 

Thank you for this PR. Are you a native Czech speaker? 
I tried my best to verify this with Google translate and it looks okey. 

Do you know a native that could verify this?

**Comment by shubhalgupta** [comment]
Yes, I am a native speaker!

**Comment by shubhalgupta** [comment]
@Nyholm please do merge this!

**Review by Nyholm — APPROVED** [review]
Lovely. =)

I'll be happy to approve this. A second maintain will review and merge this if they like this as much as I do. 

Thank you for your work!

**Comment by shubhalgupta** [comment]
@Nyholm who is the second maintainer?

**Comment by shubhalgupta** [comment]
And thanks for your approval @Nyholm

**Comment by Nyholm** [comment]
> @Nyholm who is the second maintainer?

There are a few of us. See https://symfony.com/doc/current/contributing/code/core_team.html
Im sure someone will look at this in the next 2-3 days.

**Comment by shubhalgupta** [comment]
@fabpot, @nicolas-grekas  kindly review this!

**Review by stof — APPROVED** [review]

**Comment by shubhalgupta** [comment]
@stof can you merge this, please!

**Comment by shubhalgupta** [comment]
@Nyholm can you please get it merged?

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (OskarStark) [inline src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]:
```suggestion
            <trans-unit id="104">
                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
                <target>Název souboru je příliš dlouhý. Měl by obsahovat {{ filename_max_length }} znak nebo méně.|Název souboru je příliš dlouhý. Měl by obsahovat {{ filename_max_length }} znaky nebo méně.</target>
            </trans-unit>
            <trans-unit id="105">
                <source>The password strength is too low. Please use a stronger password.</source>
                <target>Síla hesla je příliš nízká. Použijte silnější heslo, prosím.</target>
            </trans-unit>
            <trans-unit id="106">
                <source>This value contains characters that are not allowed by the current restriction-level.</source>
                <target>Tato hodnota obsahuje znaky, které nejsou povoleny aktuálním úrovní omezení.</target>
            </trans-unit>
           <trans-unit id="107">
               <source>Using invisible characters is not allowed.</source>
               <target>Používání neviditelných znaků není povoleno.</target>
           </trans-unit>
           <trans-unit id="108">
               <source>Mixing numbers from different scripts is not allowed.</source>
               <target>Kombinování čísel z různých písem není povoleno.</target>
           </trans-unit>
           <trans-unit id="109">
               <source>Using hidden overlay characters is not allowed.</source>
               <target>Použití skrytých překrývajících znaků není povoleno.</target>
           </trans-unit>
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please fix the indentation, thanks

**Comment by shubhalgupta** [comment]
@OskarStark now kindly check!

**Comment by xabbuh** [comment]
I have asked a Czech colleague to review the changes

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (jan-simbera) [inline src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]: Hi, the correct wording should be: Měl by obsahovat {{ filename_max_length }} znaků nebo méně. Same for the filename length, but it could vary, as for 2,3,4 it should be znaky, for the others znaků and for 1 znak.

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (jan-simbera) [inline src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]: The correct wording should be: Tato hodnota obsahuje znaky, které nejsou povoleny aktuální úrovní omezení.

**Review by jan-simbera — CHANGES_REQUESTED** [review]
Hi, there are some changes in the wording of the Validation Error Messages, which should be implemented to be correct in Czech.

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (stof) [reply src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]: Well, the varying part is supported by Symfony: for Czech, the Symfony translation format expects 3 variants for pluralization, not 2.

**Comment by jan-simbera** [comment]
Hi, i have corrected the pluralization as i forgot it is also different for
1. So we have 1 - znak, 2,3,4 - znaky and 5+ znaků.

Sorry, missed thats special case in the first part.

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]: @jan-simbera thanks for the review. Is that what you mean (znaků being > 4)?

```suggestion
                <target>Název souboru je příliš dlouhý. Měl by obsahovat {{ filename_max_length }} znak nebo méně.|Název souboru je příliš dlouhý. Měl by obsahovat {{ filename_max_length }} znaky nebo méně.|Název souboru je příliš dlouhý. Měl by obsahovat {{ filename_max_length }} znaků nebo méně.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf
Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.cs.xlf]:
```suggestion
                <target>Tato hodnota obsahuje znaky, které nejsou povoleny aktuální úrovní omezení.</target>
```

**Comment by nicolas-grekas** [comment]
Thank you @shubhalgupta.

**Pull request merged**
